modules/centralized-logging/variables.tf

```diff
@@ -40,6 +40,11 @@ variable "bucket_name" {
   type        = string
 }
 
+variable "parent_folder" {
+  description = "The folder id that will be logged."
+  type        = string
```

Comment:
what happens if I put an "" (empty) parent folder id as input?  (or all the projects are not in the same folder)

In other words, should we enforce any pre-condition check(s).  I think this is sorta coupled to the new boolean introduced.  This also seems to assume that all the projects will be in the same folder (i.e the governance, non-confidential, confidential, etc)

Reply:
The centralized-logging module is not a public module. We built it to be used only in the standalone examples. 

We do not consider those corner cases in the test.

Do you think that is an important check to do even in a private module?

Reply:
+1 good reminder - not as important due to the coupling for now.